root/entity/edits.tt

```diff
@@ -11,7 +11,7 @@
 
     SET layout_title = all_edits ? l('Edits for {name}', { name => name_for_title }) : l('Open Edits for {name}', { name => name_for_title });
     SET heading = all_edits ? l('Edits for {name}', { name => link_for_heading }) : l('Open Edits for {name}', { name => link_for_heading });
-    SET subheading = html_escape(entity.l_type_name) or format_entity_type_name(ent_type);
+    SET subheading = html_escape(entity.l_type_name or format_entity_type_name(ent_type));
```

Comment:
Vaguely related; missing encoding for the type name (but it’s for the heading, not the title, and current type names don’t contain special characters).